source/core/owt_base/VideoFrameConstructor.h

```diff
@@ -43,7 +50,10 @@ class VideoFrameConstructor : public erizo::MediaSink,
     };
 
     VideoFrameConstructor(VideoInfoListener*, uint32_t transportccExtId = 0);
-    VideoFrameConstructor(VideoFrameConstructor*, VideoInfoListener*, uint32_t transportccExtId = 0);
+    VideoFrameConstructor(std::shared_ptr<rtc_adapter::RtcAdapter>,
+                          VideoInfoListener*,
+                          uint32_t transportccExtId = 0);
```

Comment:
any reason the tccExId must be passed to RTCP handling at C++ layer?

Reply:
webrtc need extension ID to know pass the transport cc value in rtp extension